### ISSUE TITLE
Math Rendering Error

### ISSUE THREAD

**Issue by RoyiAvital** [mike-ward/Markdown-Edit#156]
I used Display Math to render the following:

`$$ {\left\| x \right\|}_{TV} = \sum_{i = 1}^{m - 1} \sum_{j = 1}^{n - 1} {x}_{i, j} $$`.

Yet the Math won't be rendered.

On [StackEdit](https://stackedit.io) it works perfectly (Hence it means it is not MathJax issue).

Thank You.

**Comment by RoyiAvital** [comment]
I can also add another example for Inline Math:

`\\( \mathcal{C} = \mathcal{{B}_{l, u}} = \left\{ x \mid l \leq {x}_{i, j} \leq u, \; \forall i, j \right\} \\)`

**Comment by mike-ward** [comment]
`\left\` and `\right\` are not recognized by the MathJax parser in Markdown Edit. Might be an option in MathJax I don't have enabled. I don't know enough about MathJax to say what that is. Maybe a MathJax export can chime in?

On stuff like this, screen shots are helpful.

**Comment by RoyiAvital** [comment]
Hi,

I looked at the documentation of MathJax.
Specifically all the settings (http://docs.mathjax.org/en/latest/options/tex2jax.html).
There is nothing about that actually it seems to support it according to the [commands page](http://docs.mathjax.org/en/latest/tex.html) (See at the bottom in `L` and 'R').

Here is a screenshot:

![image](https://cloud.githubusercontent.com/assets/7347975/15909134/9fb7609c-2dcd-11e6-9b0e-79d2b72566cd.png)

It seems to mark the Under Score by red.

Since all is rendered perfectly in StackEdit I don;t think it has to do with MathJax.

Is there a way to access where the MathJax script it loaded and configure it manually?

For instance, I really prefer `$...$` and it will be more compatible with other Mark Down editors.

**Comment by RoyiAvital** [comment]
OK,
I think I have an idea.

The program does something to the character `\` inside math blocks.
For instance, in LaTeX in order to create a new line one should use `\\` yet in MDE one must use `\\\` like it would drop one of the `\`.

Could it be?

**Comment by mike-ward** [comment]
I added a command line option to the Pandoc processor to escape the math portions.
```
$$a^2 + 4ab + c$$

$a^2 + 4ab + c$

\\( a^2 + 4ab + c \\)

$$ {\left\| x \right\|}_{TV} = \sum_{i = 1}^{m - 1} \sum_{j = 1}^{n - 1} {x}_{i, j} $$

```
All render correctly.

I could not get

```
\\( \mathcal{C} = \mathcal{{B}_{l, u}} = \left\{ x \mid l \leq {x}_{i, j} \leq u, \; \forall i, j \right\} \\)
```
to render. For the record, when I paste it into Slack Edit, it does not render there either.

**Comment by mike-ward** [comment]
Syntax highlighting in the editor uses the CommonMark abstract tree which knows nothing about math constructs. Coloring there will be wonky.

**Comment by RoyiAvital** [comment]
Hi,

StackEdit only uses `$...$` and `$$...$$` to resemble the work in LaTeX.

So `\\( \mathcal{C} = \mathcal{{B}_{l, u}} = \left\{ x \mid l \leq {x}_{i, j} \leq u, \; \forall i, j \right\} \\)` should become `$ \mathcal{C} = \mathcal{{B}_{l, u}} = \left\{ x \mid l \leq {x}_{i, j} \leq u, \; \forall i, j \right\} $` and it does work:

![image](https://cloud.githubusercontent.com/assets/7347975/15918657/ae91e6d4-2e11-11e6-83c2-f2856e5f29b3.png)

Thank You.

**Comment by mike-ward** [comment]
Submitted an issue to Pandoc (#2976)

**Comment by mike-ward** [comment]
From Pandoc.

Don't leave a space after the $ and it will work correctly.

---
> pandoc --mathjax
$space$
<p><span class="math">\(space\)</span></p>
> pandoc --mathjax
$ space $
<p>$ space $</p>
```

**Comment by RoyiAvital** [comment]
Hi,

Is it fixed in 1.28?

Thank You.

**Comment by mike-ward** [comment]
I did publish a fix in 1.28 but I found a few more issues so don't pull the trigger just yet on new bugs.

**Comment by RoyiAvital** [comment]
Great...

I wish Pandoc would remove the need for spaces.

Thank You.
